src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier supplier = new ColorSupplier();
+        int index = new Random().nextInt(100);
+        Ball ball = new Ball();
+        ball.setNumber(index);
+        ball.setColor(supplier.getRandomColor());
+        return ball;
+    }
```

Comment:
[All magic numbers in your code should be constants.](https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery#all-magic-numbers-in-your-code-should-be-constants)
We can add fields with type `Random` and `ColorSupplier` to our class, so we won't need to create instances every time

also, it is better to create valid instances of a `Ball` class, so we can add a constructor with appropriate arguments to the `Ball`